docs/dev/README.md

```diff
@@ -42,7 +42,8 @@ https://rust-lang.zulipchat.com/#narrow/stream/185405-t-compiler.2Frust-analyzer
 * [S-actionable](https://github.com/rust-analyzer/rust-analyzer/issues?q=is%3Aopen+is%3Aissue+label%3AS-actionable) and
   [S-unactionable](https://github.com/rust-analyzer/rust-analyzer/issues?q=is%3Aopen+is%3Aissue+label%3AS-unactionable)
   specify if there are concrete steps to resolve or advance an issue. Roughly, actionable issues need only work to be fixed,
-  while unactionable ones are effectively wont-fix. Each triaged issue should have one of these labels.
+  while unactionable ones are blocked either on user feedback (providing a reproducible example), or on larger architectural 
+  work or decisions. Each triaged issue should have one of these labels.
```

Comment:
Let’s add something a-la “it might also be the case that there’s some straightforward way to fix the issue, which we fail to see”.

On a meta note, I am now having doubts that actionable/unactionable classification creates value? I personally don’t actually use this label as a filter regularly, so it doesn’t help much with signal/noise ratio for me personally.

Reply:
How about now?

> On a meta note, I am now having doubts that actionable/unactionable classification creates value?

Yeah, I know... I don't know how useful the labels are, but I guess that's another discussion to be had.

Reply:
perfect, thanks! I like the prescriptive vs descriptive framing!

(for Russian linguistic jokes connoisseurs: 

давайте одевать одежду
давайте звонит говорить
а на прескриптивистов будем
ложить

[source](https://avva.livejournal.com/2748021.html))

bors r+